src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -289,7 +295,7 @@ public Optional<NetworkModificationResult> duplicateModifications(UUID targetGro
     public Optional<NetworkModificationResult> insertCompositeModifications(UUID targetGroupUuid,
                                                                             UUID networkUuid, String variantId,
                                                                             ReportInfos reportInfos, List<UUID> modificationsUuids) {
-        List<ModificationInfos> modificationInfos = networkModificationRepository.getCompositeModificationsInfos(modificationsUuids);
+        List<ModificationInfos> modificationInfos = networkModificationRepository.getCompositeModificationsContentInfos(modificationsUuids, false);
```

Comment:
ContentInfos ? (content or infos have the same meaning) you could keep only Infos

Reply:
Renamed.